### PR TITLE
PN-7444 - Download F24 

### PR DESCRIPTION
**Pull request by alessandrogelmi**
## Short description
Resolve download of F24 

## List of changes proposed in this pull request
- Download F24

## How to test
Start PF in dev mode and go to PXPX-PQZU-PHPQ-202306-M-1, try to download the last F24 file

### CODE REVIEW

File: packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx [hunk 1]
```diff
@@ -17,12 +17,15 @@ describe('NotificationPaymentRecipient Component', () => {
     f24Only: getF24Payments(payments, 0),
   };
 
+  const F24TIMER = 15000;
+
   it('should render component title and subtitle', () => {
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx:22]: In one of these test add the check that it is shown the correct f24 view (f24 only and f24 attachment)

File: packages/pn-personafisica-webapp/src/redux/notification/__test__/reducers.test.ts [hunk 2]
```diff
@@ -200,34 +195,6 @@ describe('Notification detail redux state tests', () => {
     expect(state.legalFactDownloadUrl).toEqual('');
   });
 
-  it('Should be able to fetch the pagopa document', async () => {
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/redux/notification/__test__/reducers.test.ts:203]: Restore these tests

File: packages/pn-personagiuridica-webapp/src/redux/notification/__test__/reducers.test.ts [hunk 3]
```diff
@@ -201,32 +194,6 @@ describe('Notification detail redux state tests', () => {
     expect(state.legalFactDownloadUrl).toEqual('');
   });
 
-  it('Should be able to fetch the pagopa document', async () => {
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/redux/notification/__test__/reducers.test.ts:204]: Restore these tests

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx (hunk 1 above)
Comment (alessandrogelmi) [reply packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx:22]: There's already a test on NotificationPaymentF24Item.test.tsx

Comment (AndreaCimini90) [reply packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx:22]: yes. here check only that, based on payment structure, it is shown the correct view

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**